### PR TITLE
[3104] SPIKE: Populating integration test data

### PR DESCRIPTION
**Pull request by asmega**
Co-authored-by: De Fong <de_fong2k@yahoo.com>
Co-authored-by: Phil Lee <asmega@gmail.com>

### Context
SPIKE: Populating integration test data

### Changes proposed in this pull request
Populating integration test data

Added 26 providers and each providers 12 courses.

### Guidance to review
Should this even go into master?

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by asmega** [comment]
[SPIKE: Populating integration test data](https://trello.com/c/aGF1tGYT/3104-spike-populating-integration-test-data)

**Comment by asmega** [comment]
[Setup benchmarking / performance test](https://trello.com/c/lyS6VGel/3033-setup-benchmarking-performance-test)

File: db/integration_seeds.rb [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      RecruitmentCycle.current_recruitment_cycle
+    end
+
+    def create_user(attributes)
+      User.find_by(attributes) || FactoryBot.create(:user, attributes)
+    end
+  end
+end
+
+if $0.end_with? "rails"
```
Comment (asmega) [inline db/integration_seeds.rb:113]: why is this here? can we not invoke this explicitly?

Comment (asmega) [reply db/integration_seeds.rb]: I've now removed this

Comment (misaka) [reply db/integration_seeds.rb]: Ya, there was a reason this was here, but I don't remember what it was now, probably should've added a comment. 🤷
